.github/workflows/third_party_scan.yml

```diff
@@ -32,19 +26,29 @@ jobs:
         uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c
         with:
           python-version: '3.7.7' # install the python version needed
-      - name: "extract and flatten deps"
-        run: python ci/deps_parser.py
-      - name: "scan deps for vulnerabilities"
-        run: python ci/scan_flattened_deps.py
-      # Upload the results as artifacts.
-      - name: "Upload artifact"
-        uses: actions/upload-artifact@c7d193f32edcb7bfad88892161225aeda64e9392
-        with:
-          name: SARIF file
-          path: osvReport.sarif
-          retention-days: 5
-      # Upload the results to GitHub's code scanning dashboard.
-      - name: "Upload to security tab"
-        uses: github/codeql-action/upload-sarif@cdcdbb579706841c47f7063dda365e292e5cad7a
+      - name: "extract, flatten, and convert to osv-scanner deps"
+        run: python ci/deps_parser.py -o osv-scanner-deps.json
```

Comment:
I'm not remembering if there's a reason that `ci/deps_parser.py` and `ci/scan_deps.py` are two separate scripts. Would it be simpler to combine them into one script?

Reply:
I'm not sure. My inclination would be to say that any issues this PR has will be easier to track down after simplifying as much as possible beforehand. But if the code in this PR is working as-is, then simplifying in a follow up PR makes sense.